config/templates/operator.yaml.in

```diff
@@ -14,6 +14,18 @@
 #
 # {{ .GeneratedWarning }}
 #
+{{- /*
+  Patch the namespace with a label so we can use a namespaceSelector for the webhooks. This ensures that webhooks don't
+  mutate/validate across namespaces.
+*/}}
+---
+apiVersion: v1
```

Comment:
File an issue please for @taroface to update the documentation when deploying the operator in another namespace, specifically with EKS and GKE.